scripts/ci.bash

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+#
+# Usage: $ CI_MACHINE_TYPE='skylake-2x' bash scripts/ci.bash
+#
+set -ex
+
+# Clean-up old results
+rm -rf gh-pages
+NEXMARK_CSV_FILE='nexmark_results.csv'
+rm -f ${NEXMARK_CSV_FILE}
+
+# Run benchmarks
+cargo bench --bench nexmark --features with-nexmark -- --first-event-rate=1000000 --max-events=100000 --cpu-cores 8  --num-event-generators 6 --source-buffer-size 10000 --input-batch-size 40000 --csv
```

Comment:
This is without persistence, right?  In that case, we want `--first-event-rate=10000000` (10M).

Reply:
Yes, I'll increase it, left it low for now so it completes fast

Reply:
Fixed